bin/oref0-setup.sh

```diff
@@ -255,6 +255,32 @@ function add_to_crontab () {
     (crontab -l; crontab -l |grep -q "$1" || echo "$2" "$3") | crontab -
 }
 
+function request_stop_local_binary () {
+    if [[ -x /usr/local/bin/$1 ]]; then
+        if pgrep -x $1 > /dev/null; then
+            read -p "Need to stop $1 to complete installation. OK? [Y]N" -r
```

Comment:
Would it be better to use the common prompt_yn functions?

Reply:
Agreed and updated!